From: Nick Rosbrook <enr0n@ubuntu.com>
Date: Fri, 27 Mar 2026 14:22:05 -0400
Subject: tmpfiles: remove duplicate /run/lock definition

On Debian, /run/lock is defined by /usr/lib/tmpfiles.d/debian.conf so
that the permissions can be adjusted. This currently results in errors
like:

 /usr/lib/tmpfiles.d/legacy.conf:14: Duplicate line for path "/run/lock", ignoring.

on package install, upgrade, etc. Remove the legacy.conf definition to
silence these warnings. We should be able to drop this patch after
resolute.

Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/2146544
Forwarded: no
---
 tmpfiles.d/legacy.conf.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tmpfiles.d/legacy.conf.in b/tmpfiles.d/legacy.conf.in
index cdef21f..7e9e531 100644
--- a/tmpfiles.d/legacy.conf.in
+++ b/tmpfiles.d/legacy.conf.in
@@ -11,7 +11,6 @@
 # by newer interfaces. Their use is discouraged on legacy-free systems. This
 # configuration is provided to maintain backward compatibility.
 
-d /run/lock 0755 root root -
 L /var/lock - - - - ../run/lock
 {% if CREATE_LOG_DIRS %}
 L$ /var/log/README - - - - ../..{{DOC_DIR}}/README.logs
